internal/docker/docker.go

```diff
@@ -249,6 +260,17 @@ func (handler *Handler) StartContainer(ctx context.Context, c config.Project, s
 	return &container, nil
 }
 
+// copyTestFiles copies the files within the container.
+func copyTestFiles(ctx context.Context, handler *Handler, containerID string, files []string, pDir string) error {
+	for _, file := range files {
+		log.Info().Str("from", file).Str("to", pDir).Msg("File copied")
```

Comment:
Maybe for better formatting we need to add spaces `after` from and `to`?

Reply:
Same as above :)

Reply:
Looks good!